docs/ci-integration/remote-buildkit.md

```diff
@@ -1,5 +1,10 @@
 # Remote BuildKit 
 
+{% hint style='info' %}
+##### Note
+This guide is related to self-hosting a remote Buildkit, however, Self-Hosted Satellites **beta** are now available. Self-Hosted Satellites provide more features, have better security, and are easier to deploy than remote Buildkit. Check out the [Self-Hosted Satellites Guide](../cloud/satellites/self-hosted.md) for more details. If your use case cannot tolerate any cloud integration, however, then self-hosting a remote Buildkit is still supported.
```

Comment:
@vladaionescu What do you think about changing "any cloud integration" to "a cloud control plane"? I feel like cloud control plane sounds less invasive than could integration. Or maybe it doesn't matter.

Reply:
Yeah, that can work. Maybe `a cloud-based control plane`? (minor clarity edit)